fe/fe-core/src/main/java/org/apache/doris/analysis/AggregateInfo.java

```diff
@@ -38,49 +38,48 @@
  * Encapsulates all the information needed to compute the aggregate functions of a single
  * Select block, including a possible 2nd phase aggregation step for DISTINCT aggregate
  * functions and merge aggregation steps needed for distributed execution.
- *
+ * <p>
  * The latter requires a tree structure of AggregateInfo objects which express the
  * original aggregate computations as well as the necessary merging aggregate
  * computations.
  * TODO: get rid of this by transforming
  *   SELECT COUNT(DISTINCT a, b, ..) GROUP BY x, y, ...
  * into an equivalent query with a inline view:
  *   SELECT COUNT(*) FROM (SELECT DISTINCT a, b, ..., x, y, ...) GROUP BY x, y, ...
- *
+ * <p>
  * The tree structure looks as follows:
  * - for non-distinct aggregation:
- *   - aggInfo: contains the original aggregation functions and grouping exprs
```

Comment:
We'd better not change the original comment indents.
You could wrap the comments with the `<pre></pre>` HTML tag, thus the indents wouldn't be modified when formatting the code style.

Reply:
done